debugging/debugging.md

```diff
@@ -13,13 +13,16 @@
 
 ### Motivation
 
-Things will go wrong during programming, even for experienced software engineers. It's important to quickly trouble-shoot code that is not behaving as expected.
+- Debugging is the art of finding errors/bugs in programs and determining how to correct them. Things will go wrong during programming, even for experienced software engineers.
+- It is ideally part of the testing process but in reality, it is done at every step of programming. Coders should debug the smallest of their modules before moving on. This decreases the number of errors thrown up during the testing phase and reduces testing time and effort significantly.
+
+**Which companies use debugging?** Everyone because all companies want their product to be bugs/errors free.
 
 ### Objectives
 **Participants will be able to:**
 - Understand the various kinds of errors that may arise
 - Use `console.log()` statements to verify logic and output at different points in the code
-- Use the rubber duck technique to talk through troublesome code
+- Use the rubber duck technique to talk through the troublesome code
```

Comment:
I don’t think “the” is necessary, but only change this if there are other corrections. No need if this is the only one.